mungegithub/mungers/blunderbuss.go

```diff
@@ -90,7 +91,7 @@ func printChance(owners weightMap, total int64) {
 	}
 }
 
-func getPotentialOwners(author string, feats *features.Features, files []*githubapi.CommitFile, leafOnly bool) (weightMap, int64) {
+func getPotentialOwners(obj *github.MungeObject, author string, feats *features.Features, files []*githubapi.CommitFile, leafOnly bool) (weightMap, int64, error) {
```

Comment:
I think that would be a better solution. The function `isAssignee` shouldn't be coupled to the MungeObject anyways.